src/main/java/com/libertymutualgroup/herman/aws/ecs/broker/cloudwatch/CloudWatchEventsBroker.java

```diff
@@ -0,0 +1,68 @@
+package com.libertymutualgroup.herman.aws.ecs.broker.cloudwatch;
+
+import com.amazonaws.services.cloudwatchevents.AmazonCloudWatchEvents;
+import com.amazonaws.services.cloudwatchevents.model.DeleteRuleRequest;
+import com.amazonaws.services.cloudwatchevents.model.DescribeRuleRequest;
+import com.amazonaws.services.cloudwatchevents.model.DescribeRuleResult;
+import com.amazonaws.services.cloudwatchevents.model.PutRuleRequest;
+import com.amazonaws.services.cloudwatchevents.model.PutRuleResult;
+import com.amazonaws.services.cloudwatchevents.model.PutTargetsRequest;
+import com.amazonaws.services.cloudwatchevents.model.PutTargetsResult;
+import com.amazonaws.services.cloudwatchevents.model.RemoveTargetsRequest;
+import com.amazonaws.services.cloudwatchevents.model.ResourceNotFoundException;
+import com.amazonaws.services.cloudwatchevents.model.RuleState;
+import com.amazonaws.services.cloudwatchevents.model.Target;
+import com.amazonaws.services.lambda.model.GetFunctionResult;
+import com.libertymutualgroup.herman.aws.lambda.LambdaInjectConfiguration;
+import com.libertymutualgroup.herman.logging.HermanLogger;
+
+public class CloudWatchEventsBroker {
+
+    private HermanLogger buildLogger;
+    private AmazonCloudWatchEvents amazonCloudWatchEvents;
+
+    public CloudWatchEventsBroker(HermanLogger buildLogger, AmazonCloudWatchEvents amazonCloudWatchEvents) {
+        this.buildLogger = buildLogger;
+        this.amazonCloudWatchEvents = amazonCloudWatchEvents;
+    }
+
+    public void brokerScheduledRule(LambdaInjectConfiguration configuration, GetFunctionResult output) {
+        if (configuration.getScheduleExpression() != null) {
+            this.buildLogger.addLogEntry("Brokering Scheduled Rule with schedule expression: " + configuration.getScheduleExpression());
+            PutRuleRequest putRuleRequest = new PutRuleRequest()
```

Comment:
Could you include the tags from the lambda here as well? It looks like `PutRuleRequest()` has a .withTags() function on it. Unfortunately it looks like it accepts `com.amazonaws.services.cloudwatchevents.model.Tag` instead of Lambda tags, so you'll need to add a conversion in TagUtil.java.

Reply:
just pushed an update